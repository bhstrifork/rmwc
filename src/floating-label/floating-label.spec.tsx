import React from 'react';
import { render } from '@testing-library/react';
import { FloatingLabel } from './';

describe('FloatingLabel', () => {
  it('renders', () => {
    const { asFragment } = render(<FloatingLabel />);
    expect(asFragment()).toMatchSnapshot();
  });
  it('renders with float', () => {
    const { asFragment } = render(<FloatingLabel float />);
    expect(asFragment()).toMatchSnapshot();
  });
});
