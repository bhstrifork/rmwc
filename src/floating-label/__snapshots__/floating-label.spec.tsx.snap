// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FloatingLabel renders 1`] = `
<DocumentFragment>
  <span
    class="mdc-floating-label"
  />
</DocumentFragment>
`;

exports[`FloatingLabel renders with float 1`] = `
<DocumentFragment>
  <span
    class="mdc-floating-label mdc-floating-label--float-above"
  />
</DocumentFragment>
`;
